src/views/app.jsx

```diff
@@ -1,25 +1,41 @@
 import React from 'react';
+import { Dispatcher } from 'consus-core/flux';
 import AuthenticationStore from '../store/authentication-store';
+import ErrorStore from '../store/error-store';
 
 import ListenerComponent from '../lib/listener-component.jsx';
 import Omnibar from './components/omnibar.jsx';
 import Models from './pages/models.jsx';
+import ErrorModal from './components/error-modal.jsx';
+
 export default class App extends ListenerComponent {
 
     constructor() {
         super();
     }
 
     getStores() {
-        return [AuthenticationStore];
+        return [AuthenticationStore, ErrorStore];
     }
 
     getState() {
         return {
-            loggedIn: AuthenticationStore.loggedIn()
+            loggedIn: AuthenticationStore.loggedIn(),
+            errorTag: ErrorStore.getTag(),
```

Comment:
Not yet its not. I thought were going to use tags to style it pretty ala [Bootstrap alerts](http://v4-alpha.getbootstrap.com/components/alerts/#examples).